packages/picasso/src/Menu/styles.ts

```diff
@@ -18,7 +18,9 @@ PicassoProvider.override(({ shadows }: Theme) => ({
 export default () =>
   createStyles({
     root: {
-      outline: 0
+      outline: 0,
+      padding: '8px 0px 8px 0px',
+      borderRadius: '4px'
```

Comment:
```suggestion
      borderRadius: sizes.borderRadius.small
```

`export default () =>` 👉 `export default ({ sizes }) =>`

Reply:
Oh I totally missed this. Thank you.